opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -371,31 +375,18 @@ int mca_btl_tcp_endpoint_send(mca_btl_base_endpoint_t* btl_endpoint, mca_btl_tcp
 
 
 /*
- * A blocking send on a non-blocking socket. Used to send the small amount of connection
- * information that identifies the endpoints endpoint.
+ * A blocking send on a non-blocking socket. Used to send the small
+ * amount of connection information that identifies the endpoints endpoint.
  */
 static int
 mca_btl_tcp_endpoint_send_blocking(mca_btl_base_endpoint_t* btl_endpoint,
```

Comment:
since i moved mca_btl_tcp_recv_blocking to use it across multiple places. I moved mca_btl_tcp_send_blocking with the intention it will be shared too